snap/info_test.go

```diff
@@ -128,6 +128,10 @@ func (s *infoSuite) TestReadInfo(c *C) {
 }
 
 func makeTestSnap(c *C, yaml string) string {
```

Comment:
yea, it exists mostly for clients outside of snap/ itself which shouldn't need to test verification again, I might look again, anyway as I said shouldn't block this